e2e/libs/network/network.py

```diff
@@ -0,0 +1,29 @@
+from robot.libraries.BuiltIn import BuiltIn
+from utility.utility import get_control_plane_nodes
+from node_exec import NodeExec
+
+def get_latency_in_ms():
+    latency_ms_ms = int(BuiltIn().get_variable_value("${CONTROL_PLANE_NODE_NETWORK_LATENCY_IN_MS}"))
```

Comment:
The `CONTROL_PLANE_NODE_NETWORK_LATENCY_IN_MS` should reflect in the function name.

Reply:
refined.